app/decorators/controllers/accounts_controller_decorator.rb

```diff
@@ -81,12 +84,16 @@ def last_assignment_due
     account_settings_params[:last_assignment_due].present? ? account_settings_params[:last_assignment_due] : false
   end
 
+  def course_start_time
+    account_settings_params[:course_start_time].present? ? account_settings_params[:course_start_time] : false
```

Comment:
Should we be returning `false` if the desired value is not present?

Intent: I'm not sure what the correct convention is, but it seems like if the desired value is present, we're returning that object, otherwise we're returning a boolean value

Reply:
Thanks for pointing this out! We've changed this code so that if the value is not present, it will default to a value. It will default to 12:00 PM. Please confirm if that sounds good to you!

(We're actually currently changing the default value to 12:00 AM so you'll see that change soon, but the fix is there for the most part)

Reply:
The reason I was asking is because I was unsure what the right answer is, but it just seemed odd that we're returning the time value and if it's not present we're returning a different data type (boolean).  I was asking because I wasn't sure if this was a Ruby convention or not.

Reply:
Yes you were correct, we eventually changed it to "12" (midnight) to better reflect the string type. Thank you